recipes/cosolvkit/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "cosolvkit" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cosolvkit-{{ version }}.tar.gz
+  sha256: 960aa8bf95db081c62df5bb3634d11e06b4a3f28740b1400de05b7c62503d44b
+
+build:
+  skip: True  # [win or aarch64]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python ==3.10
+    - setuptools >=61.0
+    - pip
+  run:
+    - python ==3.10
+    - openmmforcefields ==0.12
+    - pdbfixer
+    - mdanalysis
+    - espaloma
+    - pymol-open-source
+    - griddataformats
+
+
+test:
+  imports:
+    - openmm
+    - espaloma
+    - cosolvkit
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/forlilab/cosolvkit
```

Comment:
I'm getting a 404 here.

Reply:
Modified the visibility and now it should be reachable.